docs/HOL_Fault-Injection-for-Azure-IaaS-with-OMS/README.md

```diff
@@ -0,0 +1,322 @@
+HOL - Fault Injection for Azure IaaS with OMS
+=============================================
+
+In this lab you will set up PartsUnlimitedMRP's environment hosted on multiple VMs in Virtual Machine Scale Set (VMSS) for better reliability. Operations Management Suite(OMS) will be used to monitor how this environment handles VM failures under the load. Excepted number of user for this system is 1200 people.
+
+## Pre-requisites:
+
+* Azure account with a subscription
+
+* VSTS account
+
+
+## Tasks Overview:
+**1. Setup OMS workspace** This task will show you how to create OMS Workspace using Azure and then find WORKSPACE ID and PRIMARY KEY used to connect to OMS.
+
+**2. Deploy ARM template** In this task you will deploy an ARM template which creates a VMSS of Linux machines and then installs MRP environment on it.
+
+**3. Set up Load Test** In this task you will configure a load test using VSTS.
+
+**4. Inject faults into VMSS** In this task you will learn how to inject faults into VMSS using Azure Portal, PowerShell script or [Chaos-Dingo](https://github.com/jmspring/chaos-dingo).
+
+**5. Analyze load test's results.** In this task you will collect the information related to the load test and fault injection and then analyze it.
+
+
+
+## HOL:
+###Task 1: Setup OMS workspace
+
+**Step 1.** Open Azure and create a new resource group for this HOL.
+
+1. Click on "Resource groups" icon, then on "+ Add".
+
+    ![](media/2.png)
+
+2. Enter a name for your resource group, pick a subscription, select a location and click on "Create".
+
+    ![](media/3.png)
+
+
+**Step 2.** Add "Log Analytics" to this resource group.
+
+1. Open the new resource group and click on "+ Add" button on the "Overview" page to add a new resource.
+
+    ![](media/4.png)  
+
+2. Filter resources by typing "Log Analytics" into the search bar, press "Enter", click "Log Analytics" and click on "Create" button.
+
+    ![](media/5.png)  
+
+3. Create an OMS workspace. Enter name, pick a subscription, location, current resource group and a pricing tier then click on "OK" button.
+
```

Comment:
Please change "current resource group" to "the resource group you created in step 1"